src/theory/arith/arith_rewriter.cpp

```diff
@@ -346,27 +347,64 @@ RewriteResponse ArithRewriter::preRewritePlus(TNode t){
 RewriteResponse ArithRewriter::postRewritePlus(TNode t){
   Assert(t.getKind() == kind::PLUS);
 
+  if (t.getNumChildren() == 1)
+  {
+    return RewriteResponse(REWRITE_DONE, t[0]);
+  }
```

Comment:
[It looks like this shouldn't be possible.](https://github.com/cvc5/cvc5/blob/c9daef3594dec48693ede0c13e7fc9804835ef14/src/theory/arith/kinds#L15)

Reply:
Good catch.